api/v1alpha1/virtualmachine_conversion.go

```diff
@@ -552,6 +634,7 @@ func Convert_v1alpha2_VirtualMachineStatus_To_v1alpha1_VirtualMachineStatus(
 	out.Phase = convert_v1alpha2_Conditions_To_v1alpha1_Phase(in.Conditions)
 	out.VmIp, out.NetworkInterfaces = convert_v1alpha2_NetworkStatus_To_v1alpha1_Network(in.Network)
 	out.LastRestartTime = in.LastRestartTime
+	out.Conditions = convert_v1alpha2_Conditions_To_v1alpha1_Conditions(out.Conditions)
```

Comment:
This should be `in.Conditions`?

Reply:
The autoConvert above already converted the types. I'll rename this from `convert_` to like `filter_` or something else.

Reply:
From the above seem like the autoconvert retains all the v1a2 Conditions as is for the v1a1 CR to use. This convert_/filter_ method is to primarily synthesize the v1a1 preReq condition then, BV?.

Reply:
Correct